Sources/Sentry/SentryPropagationContext.m

```diff
@@ -0,0 +1,50 @@
+#import "SentryPropagationContext.h"
+#import "SentryDSN.h"
+#import "SentryHub+Private.h"
+#import "SentryId.h"
+#import "SentryOptions+Private.h"
+#import "SentrySDK+Private.h"
+#import "SentryScope+Private.h"
+#import "SentrySpanId.h"
+#import "SentryTraceContext.h"
+#import "SentryTraceHeader.h"
+#import "SentryUser+Private.h"
+
+@implementation SentryPropagationContext
+
+- (instancetype)init
+{
+    if (self = [super init]) {
+        self.traceId = [[SentryId alloc] init];
+        self.spanId = [[SentrySpanId alloc] init];
+    }
+    return self;
+}
+
+- (SentryTraceHeader *)traceHeader
+{
+    return [[SentryTraceHeader alloc] initWithTraceId:self.traceId
+                                               spanId:self.spanId
+                                              sampled:kSentrySampleDecisionNo];
```

Comment:
Not sure if this is relevant, but in `sentry-java` sampled can be `true`, `false` or `null`.

Reply:
Maybe because java may run in the server and propagate whats comes from the client? I have no clue how this could have any other value here in the cocoa.

Reply:
>Wouldn't it make more sense to not even swizzle -[NSURLSessionTask resume] if network tracking is disabled, so that this isn't even a valid codepath?

Thats the thing, we want this code to run even with network tracking is disabled, we not tracking the network, just adding and extra header to the request to propagate to the server.